### DISCUSSION TITLE
Add boolean outputs to Fast Groups Bypasser/Muter

### DISCUSSION THREAD

**Discussion by eikonoklastes** [rgthree/rgthree-comfy#345] [Ideas]
Hi, I was wondering if each item in the Fast Groups Bypasser list could have their own boolean output so that we can do some basic flow control based on the state of that switch. 

For example, for a CogVideo workflow, I have a frame interpolation group set up. If that's enabled, I want my output frame rate to be 60 FPS, but if it is disabled by the Fast Groups Bypasser, I want to set the frame rate to 8 FPS. 

Many thanks.

**Comment by Explyy** [comment]
That would be amazing!!!

**Comment by loz2754** [comment]
If anyone is still wanting this feature, my AUN-Comfyui-Nodes pack has several way of accomplishing this.

<img width="1118" height="732" alt="image" src="https://github.com/user-attachments/assets/45860e0b-12f0-4803-baef-adf2598b8440" />

**Comment by BrummBearV2** [comment]
i really wish this would come with rgthree. any plans to add it?

**Comment by giminic** [comment]
Probably not relevant anymore for OP but the Any Switch can be utilized to control values based on groups Mute/Bypass status.

<img width="1951" height="805" alt="Screenshot 2026-05-16 075301" src="https://github.com/user-attachments/assets/c0c06332-2fcf-42a3-a77a-de8562005697" />

**Reply by Fembro-Brite** [reply]
yeah still good to shows there is more than one way to do things but some may not even know of that switch which makes many "all in one" approachs possible withouy having to constantly plug and change things.🤹
